ports/gstreamer/portfile.cmake

```diff
@@ -252,42 +166,25 @@ if (NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
     endif()
 endif()
 
-vcpkg_fixup_pkgconfig()
-
-# Remove duplicated GL headers (we already have `opengl-registry`)
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/include/KHR"
-                    "${CURRENT_PACKAGES_DIR}/include/GL"
-)
-file(RENAME "${CURRENT_PACKAGES_DIR}/lib/gstreamer-1.0/include/gst/gl/gstglconfig.h"
-            "${CURRENT_PACKAGES_DIR}/include/gstreamer-1.0/gst/gl/gstglconfig.h"
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/lib/${pkg_name}/pkgconfig"
```

Comment:
Why remove this folder?

Reply:
At this point, they are copied from `lib/gstreamer-1.0/pkgconfig` to `lib/pkgconfig`. The folder will be redundant.